packages/terra-form-field/src/Field.jsx

```diff
@@ -132,7 +141,7 @@ const Field = (props, { intl }) => {
   );
 
   return (
-    <div {...customProps} className={fieldClasses}>
+    <div style={customStyles} {...customProps} className={fieldClasses}>
```

Comment:
I think style needs to be declared after custom properties to prevent maxWidth from being overwritten. 

You can double check before making the change with something like this:
```
<Field maxWidth="300px" style={{ color: 'red'}} />
```

```suggestion
    <div {...customProps}  style={customStyles} className={fieldClasses}>
```

Reply:
It seems to apply maxWidth regardless of order (either swapping around the props on the test case, swapping style / customProps on the component, or both).

Reply:
That makes sense, style was deconstructed so it doesn't get included with the customProps spread.